src/plugins/intel_cpu/src/nodes/kernels/riscv64/cpu_isa_traits.cpp

```diff
@@ -29,7 +29,7 @@ struct RVVGenerator : public CodeGenerator {
 };
 
 // NOLINTBEGIN(misc-include-cleaner) bug in clang-tidy
-thread_local sigjmp_buf jmpbuf;
+thread_local sigjmp_buf jmpbuf;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
```

Comment:
can we also make it static within a function?

Reply:
Done